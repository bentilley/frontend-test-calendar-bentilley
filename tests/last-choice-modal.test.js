import React from 'react';
import renderer from 'react-test-renderer';

import LastChoiceModal from 'components/num-tickets/last-choice-modal';

describe('Showing component', () => {
  it('renders correctly: nothing selected', () => {
    const component = renderer.create(<LastChoiceModal
                                        number={6}
                                        numAdditionalOptions={4}
                                      />);
    const instance = component.getInstance();
    const result = instance.render();
    expect(result).toMatchSnapshot();
  });
  it('renders correctly: one selected', () => {
    const component = renderer.create(<LastChoiceModal
                                        number={6}
                                        numAdditionalOptions={4}
                                        currentSelection={7}
                                      />);
    const instance = component.getInstance();
    const result = instance.render();
    expect(result).toMatchSnapshot();
  });
});
