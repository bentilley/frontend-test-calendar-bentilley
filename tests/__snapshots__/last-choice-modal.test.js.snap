// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Showing component renders correctly: nothing selected 1`] = `null`;

exports[`Showing component renders correctly: one selected 1`] = `null`;
